features/dual-stack-support.md

```diff
@@ -0,0 +1,229 @@
+
+[//]: # (The primary lead or leads responsible for the feature. These individuals serve as a point of contact for the feature.)
+
+
+**Short description:**
+
+Add support for Dual Stack network model in Istio. With this, besides both IPv6 and IPv4 network model, Istio now supports dual-stack at the moment.
+
+[//]: # (A short description of the feature. One or two sentences maximum.)
+
+
+**Design Docs:**
+
+[Optimized design solution for Dual Stack support in Istio](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing)
+
+[//]: # (Design docs for feature)
+
+
+**Relevant Documentation:**
+
+[Dual Stack support](https://github.com/istio/istio/issues/40394)
+
+[//]: # (Links to relevant documentation for feature)
+
+**RFC:**
+
+[Original design solution for Dual Stack support in Istio](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing)
+
+[//]: # (Link to RFC for feature)
+
+
+---
+
+## Experimental
+
+### Requirements:
+
+[//]: # (All information in this section is mandatory for promotion. Please modify the links in this)
+[//]: # (section.)
+
+- [x] [User stories](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing) reviewed in a work group meeting.
+
+[//]: # (User stories are a way to communicate user value. User stories follow the style)
+[//]: # (as a [type of user], I want [an action] so that [a benefit/a value]. Istio currently has no user)
+[//]: # (story template. Maybe you can make one?)
+
+[//]: # (User stories must be presented in a work group meeting. They need no approval and are later integrated)
+[//]: # (into the RFCs, which do need approval for alpha. You may find value to negotiate within the work group where the)
+[//]: # (user stories are presented to help clarify the user stories.)
+
+- [x] [RFC Authored] - [Supporting Dual Stack](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing).
+
+[//]: # (An RFC is mandatory to graduate to experimental. The RFC does not have to be reviewed in a work group)
+[//]: # (meeting to graduate to experimental.)
+
+- [x] [Documentation](https://istio.io/latest/news/releases/1.17.x/announcing-1.17/#istio-dual-stack-support), as well as [Supporting Dual Stack](https://docs.google.com/document/d/15LP2XHpQ71ODkjCVItGacPgzcn19fsVhyE7ruMGXDyU/edit?usp=sharing) for enabling and using the feature.
+
+[//]: # (The documentation instructions may exist on the developer wiki or the team drive. They may include instructions)
+[//]: # (for building running a `istioctl experimental command`, or using the preview profile,)
+[//]: # (or any other relevant information.)
+
+- [x] [Feedback plan](insert_your_link_here). N/A, we already have feedback from users that this is desired and useful.
+
+[//]: # (This may include user feedback meetings, discuss.istio.io conversations, GitHub issues, or mailing lists.)
+
```

Comment:
Maybe include use cases for which users have requested this feature

Reply:
Hi @johnma14, I added some requirement and discussion issues in this section.